TUM Campus App/PlansTableViewCell.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  PlansTableViewCell.swift
+//  TUM Campus App
+//
+//  Created by Florian Gareis on 14.12.16.
+//  Copyright © 2016 LS1 TUM. All rights reserved.
+//
+
+import UIKit
+
+class PlansTableViewCell: UITableViewCell {
+
```

Comment:
Not really that important but there's an extra free line here. It's bugging the hell out of me